replace-B2G.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+if [[ ! -d "$B2G_PATH" ]]; then
+  B2G_PATH=../B2G
+fi
+
+make install
+make sync
+GAIA_DEV_PIXELS_PER_PX=2.25 make shallow
```

Comment:
@KevinGrandon 
How about we require this from the shim script here? https://bugzilla.mozilla.org/show_bug.cgi?id=1141191#c1